sources/packages/forms/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN ln -sf $NPM_PACKAGES/node_modules node_modules
 ENV DEBUG=""
 
 RUN set -x \
-  && chmod -R 777 /app/
+  && chmod -R 770 /app/
```

Comment:
Shouldn't the group have only read and execute permissions?

Reply:
yes i tried with 750, but it didnt start the application. The write is required for the group to create users in the server, i guess. The tried combinations are 640, 644, 650, 654, 750, 754, 770, 774 and 777. So with the least permission level of 770, 774 and 777 which worked. Selected 770.